upgrade-plugins/src/main/java/org/exoplatform/ecms/upgrade/templates/NodeTypeTemplateUpgradePlugin.java

```diff
@@ -67,6 +76,24 @@ public void processUpgrade(String oldVersion, String newVersion) {
       log.info("Start " + this.getClass().getName() + ".............");
     }
     String unchangedNodeTypes = PrivilegedSystemHelper.getProperty("unchanged-nodetype-templates");
+    String previousPlfVersion = PRODUCT_VERSION_ZERO;
+    Set<String> modifiedTemplateLog = new HashSet<String>();
+    try {
+      modifiedTemplateLog = Utils.getAllEditedConfiguredData(TemplateServiceImpl.class.getSimpleName(),
```

Comment:
modifiedTemplateLog does not seem intuitive as a name. modifiedTemplates ?
<= modifiedTemplateLog mean Templates those were modified and mark in Log data.
So I prefer to use name "modifiedTemplateLog"